docs/src/contributing_apub_api_outline.md

```diff
@@ -1,380 +1,592 @@
 # Activitypub API outline
 
-**This document is completely outdated and doesn't reflect the current state of Lemmy at all**
-
-- Start with the [reddit API](https://www.reddit.com/dev/api), and find [Activitypub vocab](https://www.w3.org/TR/activitystreams-vocabulary/) to match it.
-
-<!-- toc -->
```

Comment:
Keep this toc and tocstop, then use markdown-toc:

```
npm i -g markdown-toc
markdown-toc -i contributing_apub_api_outline.md
```

This installs markdown-toc and runs it in place. Its good to have a TOC on a largo hierarchical doc like this.

Reply:
Nice, done.